pytorch_lightning/utilities/imports.py

```diff
@@ -54,4 +54,4 @@ def _module_available(module_path: str) -> bool:
 _GROUP_AVAILABLE = platform.system() != 'Windows' and _module_available('torch.distributed.group')
 _FAIRSCALE_PIPE_AVAILABLE = _FAIRSCALE_AVAILABLE and LooseVersion(torch.__version__) >= LooseVersion("1.6.0")
 _BOLTS_AVAILABLE = _module_available('pl_bolts')
-_PYTORCH_PRUNE_AVAILABLE = _module_available('torch.nn.utils.prune')
+_PYTORCH_GREATER_EQUAL_1_6_0 = LooseVersion(torch.__version__) >= LooseVersion("1.6.0")
```

Comment:
This file need to be changed to the new naming:
https://github.com/PyTorchLightning/pytorch-lightning/blob/26cc3b5357ce08c2851f6abe8ff090121c5d2dd2/pytorch_lightning/callbacks/pruning.py#L31

Reply:
Sorry, I'm not sure about what do you mean. Can you elaborate?